src/routes/Liquidity/Liquidity.module.scss

```diff
@@ -0,0 +1,7 @@
+.liquidity {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  min-width: 500px;
+  transform: translate(-50%, -50%);
+}
```

Comment:
You have all those css settings in `center` class, why not reuse it?
All you need here is probably `min-width: 500px;`

Reply:
you right